control.py

```diff
@@ -0,0 +1,147 @@
+"""
+Contains useful commands to work with the bot and the server. Used by make tool.
+It relies on some linux-specific logic, and thus might not work in another OS.
+"""
+
+import sys
+import time
+import argparse
+from server.read_config import config
+from ai_replica.engine.reconstruct_mind import reconstruct
+from ai_replica.utils.system import (
+  execute_command,
```

Comment:
The indentation is not a multiple of 4, which is against [PEP 8](https://peps.python.org/pep-0008/).

I would suggest to apply [Black](https://github.com/psf/black) to automatically fix such problems in the entire code base. Personally, I'm using it for all my collaborative Python projects as it mitigates all code-style-related issues.

Reply:
Indentation fixed.